src/materials/ConcreteThermalMoisture.C

```diff
@@ -66,8 +66,6 @@ ConcreteThermalMoisture::validParams()
   params.addParam<Real>("aggregate_mass", 1877.0, "aggregate mass (kg) per m^3");
   params.addParam<Real>("water_to_cement_ratio", 0.43, "water to cement ratio");
   params.addParam<Real>("concrete_cure_time", 23.0, "concrete curing time in days");
-  params.addDeprecatedParam<Real>(
-      "aggregate_vol_fraction", 0.7, "volumetric fraction of aggregates", "Not used");
```

Comment:
This is failing the ASR assessment cases.

Reply:
Yeah, I'll remove that commit from this PR.  It's because there hasn't been a successful update of the blackbear submodule in grizzly for a while. I almost have that fixed.